draft-ietf-quic-invariants.md

```diff
@@ -352,7 +352,7 @@ The following statements are NOT guaranteed to be true for every QUIC version:
 * QUIC uses an AEAD (AEAD_AES_128_GCM {{?RFC5116}}) to protect the packets it
   exchanges during connection establishment
 
-* QUIC packet numbers appear after the Version field
+* QUIC packet numbers appear after the connection ID fields
```

Comment:
This statement is bit weird, as the packet number field of an Initial packet does not immediately follow the connection ID fields. The field appears after the token field. And I point this out in relation to Initial packets, because IIUC the primary intended reader here are the middlebox developers, who would have the possibility of analyzing Initial packets, but no other packet types with a packet number field.

Reply:
Of course.  I was trying to be economical with language, but clearly that needs to be better.